lib/pharos/salt_minion.rb

```diff
@@ -30,9 +30,9 @@ def roles?
     end
 
     # Assign role to this minion.
-    def assign_role(role:)
-      append_grain key: "roles", val: ROLES_MAP[role]
-      role
+    def assign_role(new_role)
```

Comment:
Here I'd just call the parameter `role`. This semantics are a bit different than the one in the `Minion` class, where it has a `role` and we are assigning a `new_role`, here we could just keep this as `def assign_role(role)`.